contracts/metagov/VeBalHelper.sol

```diff
@@ -141,11 +166,28 @@ contract VeBalHelper is Ownable {
     /// @notice Withdraw ERC20 tokens that are on the pcvDeposit
     /// @dev this will be needed to withdraw B-80BAL-20WETH after exitLock(),
     /// but also to withdraw BAL and bb-a-usd earned from protocol fees
-    function withdrawERC20(
+    function withdrawERC20fromPCV(
         address token,
         address to,
         uint256 amount
     ) external onlyOwner {
         pcvDeposit.withdrawERC20(token, to, amount);
     }
+
+    function withdrawETHfromPCV(address to, uint256 amount) external onlyOwner {
+        pcvDeposit.withdraw(to, amount);
```

Comment:
this needs to be withdrawETH

Reply:
Yep, needs to implement the PCVDeposit interface on our side

Reply:
ty!